contracts/utils/VoteCounting.sol

```diff
@@ -123,6 +105,30 @@ contract VoteCounting is GovSharedStorage {
         );
     }
 
+    function setUserHasBeenRewarded(
```

Comment:
modifier?

Reply:
No, it's used when the user withdraw his rewards